modules/persistent-storage-csi-smb-cifs-create-sc.adoc

```diff
@@ -0,0 +1,38 @@
+// Module included in the following assemblies:
+//
+// * storage/container_storage_interface/persistent-storage-csi-smb-cifs.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="persistent-storage-csi-smb-cifs-create-sc_{context}"]
+= Creating a storage class for SMB CIFS
+
+After installing the Operator, you should create a storage class for dynamic provisioning of Server Message Block (SMB) protocol Common Internet File System (CIFS) dialect volumes.
+
+.Prerequisites
+* You are logged in to the running {product-title} cluster.
+
+.Procedure
+To create a storage class:
+
+* Create a storage class using the following example YAML file:
++
+[source,yaml]
+.Example YAML file
+--
+apiVersion: storage.k8s.io/v1
+kind: StorageClass
+metadata:
+  name: samba
+provisioner: smb.csi.k8s.io
+parameters:
+  source: //samba-server.samba-server.svc.cluster.local/share
+  csi.storage.k8s.io/provisioner-secret-name: smbcreds
+  csi.storage.k8s.io/provisioner-secret-namespace: samba-server
+  csi.storage.k8s.io/node-stage-secret-name: smbcreds
+  csi.storage.k8s.io/node-stage-secret-namespace: samba-server
```

Comment:
These four lines assume that the user put username/password for access to Samba sever to a Secret with name `smbcreds` in the namespace `samba-server`:

```
apiVersion: v1
kind: Secret
metadata:
  name: smbcreds
  namespace: samba-server
stringData:
  username: <put_username_here>
  password: <put_password_here>
```

I think we need such an explanation about `smbcreds` and `samba-server` to make the doc useful.

Reply:
+1 we need a more generic snippet and detailed explanations below